File: FrontEnd/Admin/user creation/Payment.js
class Payment {
    paymentDate;
    amount;
    details;
    memberId;
    constructor(amount, details,memberId) {
        const date = new Date();
        this.paymentDate = `${date.getFullYear()}-${date.getMonth() + 1}-${date.getDate()}`
        this.amount = amount;
        this.details = details;
        this.memberId=memberId;
    }


}

File: GYM_API/GYM_API/Entities/Payment.cs
namespace GYM_API.Entities
{
    public class Payment
    {
        public int Id { get; set; }
        public DateTime PaymentDate { get; set; }
        public decimal Amount { get; set; }
        public string Details { get; set; }
        public int MemberId { get; set; }

    }
}
